projects/ngx-permissions/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Hi. Why Ivy is disabled?

Reply:
Is not yet recommended by the Angular team.
According to [this table](https://gist.github.com/LayZeeDK/61caba93df1ec1a0788c94a973c8dfac#file-angular-view-engine-to-ivy-transition-csv) Ivy should be enabled with Angular 10